src/main/java/net/helpscout/api/ApiDeclaration.java

```diff
@@ -0,0 +1,47 @@
+package net.helpscout.api;
+
+import net.helpscout.api.model.*;
+import net.helpscout.api.model.customer.SearchCustomer;
+import retrofit2.Call;
+import retrofit2.http.*;
+
+import java.util.Map;
+
+/**
+ * @Author: ivan
+ * Date: 03.09.16
+ * Time: 14:03
+ */
+public interface ApiDeclaration {
+
+    //    @Headers({
+    //            "Content-Type: application/json;charset=utf-8",
+    //            "Accept: application/json"
+    //    })
+    @GET("customers.json")
+    Call<Page<Customer>> getCustomers(@Query("page") Integer page, @QueryMap Map<String, String> searchParams, @Query("fields") String... fields);
```

Comment:
As mentioned in PR comment, with `Feign` you cold do just
`Page<Customers> getCustomers(....)`